src/i18n/alias.ts

```diff
@@ -4,8 +4,10 @@ import { detectLocale, type LocaleDetector } from 'typesafe-i18n/detectors';
 
 export const LOCALE_ALIAS = new Map<string, Locales>([
   ['zh-CN', 'zh-HanS'],
+  ['zh-SG', 'zh-HanS'],
   ['zh-TW', 'zh-HanT'],
   ['zh-HK', 'zh-HanT'],
+  ['zh-MO', 'zh-HanT'],
   ['zh', 'zh-HanS'],
```

Comment:
Actually the generic way to achieve this is to check `new Intl.Locale("zh-HK").maximize().script` instead

Reply:
For your particular use case, you can write something like
```ts
() => acceptLanguageDetector().map(language => {
	const locale = new Locale(language);
	return locale.language === 'zh' ? new Locale('zh', { script: locale.maximize().script }).baseName : language;
});
```
or equivalently
```ts
() => acceptLanguageDetector().map(locale => {
	const { language, script } = new Locale(locale).maximize();
	return language === 'zh' ? new Locale(language, { script }).baseName : locale;
});
```
But this is not general enough anyway